husky_description/urdf/accessories/intel_realsense.urdf.xacro

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0"?>
```

Comment:
Why not use https://github.com/IntelRealSense/realsense-ros/tree/development/realsense2_description ?

Reply:
Fixed

Reply:
Ran into a couple of issues using the RS macros. Likely PEBKAC errors on my part. This will work as-is for now (and the RS macros introduce a lot of extra joints we don't necessarily need).
Made a note to look into tidying this up & using the official URDFs later, especially if we start supporting other RS sensors (e.g. the L515 on the Dingo/Jackal/Husky)